### PR TITLE
Optimize some logging/getting/deleting paths for ER

### PR DESCRIPTION
**Pull request by conradoverta**
- Observations
- Metrics
- Hyperparameters

### CODE REVIEW

File: backend/common/src/main/java/ai/verta/modeldb/common/monitoring/AuditLogInterceptor.java [hunk 1]
```diff
@@ -62,7 +55,7 @@ public void close(Status status, Metadata trailers) {
                 } else {
                   message = String.format(AUDIT_WAS_CALLED_WRONG, callCount, methodName);
                 }
-                LOGGER.error(message);
+                //                LOGGER.error(message);
```
Comment (ghost) [inline backend/common/src/main/java/ai/verta/modeldb/common/monitoring/AuditLogInterceptor.java]: sus?

File: backend/common/src/main/java/ai/verta/modeldb/common/monitoring/AuditLogInterceptor.java [hunk 2]
```diff
@@ -62,7 +55,7 @@ public void close(Status status, Metadata trailers) {
...
                 failed_audit_logging.labels(methodName).inc();
```
Comment (conradoverta) [inline backend/common/src/main/java/ai/verta/modeldb/common/monitoring/AuditLogInterceptor.java]:
```suggestion
                failed_audit_logging.labels(methodName).inc();
```

File: backend/common/src/main/java/ai/verta/modeldb/common/monitoring/AuditLogInterceptor.java (hunk 1 above)
Comment (conradoverta) [reply backend/common/src/main/java/ai/verta/modeldb/common/monitoring/AuditLogInterceptor.java]: Removed bc it's extremely noisy on health checks. cc @ad-47 to make health checks not print these errors.

**Review by ghost — APPROVED** [review]

File: backend/common/src/main/java/ai/verta/modeldb/common/monitoring/AuditLogInterceptor.java (hunk 1 above)
Comment (ghost) [reply backend/common/src/main/java/ai/verta/modeldb/common/monitoring/AuditLogInterceptor.java]: oh yeah that one is annoying

**Review by ghost — APPROVED** [review]

**Pull request merged**